MangoAPI.Client/src/app/components/chats/chats.component.ts

```diff
@@ -130,6 +134,16 @@ export class ChatsComponent implements OnInit, OnDestroy {
     }).catch(err => console.error(err.toString()));
   }
 
+  onJoinChatClick() : void {
+    this._userChatsService.joinCommunity(this.activeChatId).pipe(takeUntil(this.componentDestroyed$)).subscribe( {
+      next: _ => {
+        this.searchChatQuery = '';
```

Comment:
After I joined chat the button is still on the screen, change chat status to `isMember = true`

Reply:
good code, hachatur, i love u